service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -0,0 +1,48 @@
+package chartoperator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
```

Comment:
Trying that again. 😛 

```suggestion
```

Reply:
Ah, thought that you meant other lines :)